src/input/SimModSuite.h

```diff
@@ -527,6 +527,31 @@ class SimModSuite : public MeshInput {
         }
       }
     }
+    if (MeshAtt.lFaceIdNoMesh.size() > 0) {
+      for (std::list<int>::iterator it = MeshAtt.lFaceIdNoMesh.begin(); it != MeshAtt.lFaceIdNoMesh.end();
```

Comment:
Should be `for (auto& elem : MeshAtt.lFaceIdNoMesh) {` in modern C++

Reply:
good to know (and shorter to write), thx.